src/fastmath/ml/regression.clj

```diff
@@ -1246,3 +1269,61 @@
      {:dose ##Inf
       :p p
       :stderr ##Inf})))
+
+(defn- residuals-analysis
+  "Summary statistics used for printing residuals with model"
+  [residuals]
+  [{:min (apply min residuals)
+    :1q (stats/quantile residuals 0.25)
+    :median (stats/median residuals)
+    :3q (stats/quantile residuals 0.75)
+    :max (apply max residuals)}])
+
+(defn- residuals-label
+  [model]
+  (cond
+    (= model :glm) "Deviance Residuals:"
+    (= model :ols) "Weighted Residuals:"
```

Comment:
`weighted` residuals should be only used when weights are used. Without weights they are just `residuals`. So I would restate this since it could be misleading.

Reply:
Got it! I'll have that updated soon